assets/src/setup/pages/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TechnicalBackground } from './technical-background';
+import { Goals } from './goals';
+import { TemplateMode } from './template-mode';
+import { ChooseReaderTheme } from './choose-reader-theme';
+import { SiteConfigurationSummary } from './site-configuration-summary';
+import { Save } from './save';
+import { SiteScan } from './site-scan';
+
+/**
+ * Settings for the pages in the application.
+ */
+export const PAGES = [
+	{
+		title: __( 'Site scan', 'amp' ),
+		PageComponent: SiteScan,
+	},
+	{
+		title: __( 'Technical background', 'amp' ),
+		PageComponent: TechnicalBackground,
+	},
+	{
+		title: __( 'Site goals', 'amp' ),
+		PageComponent: Goals,
+	},
+	{
+		title: __( 'Template modes', 'amp' ),
+		PageComponent: TemplateMode,
+	},
+	{
+		title: __( 'Reader themes', 'amp' ),
+		PageComponent: ChooseReaderTheme,
+	},
+	{
+		title: __( 'Site configuration', 'amp' ),
```

Comment:
Maybe just “Summary” would be more concise. 

```suggestion
		title: __( 'Summary', 'amp' ),
```